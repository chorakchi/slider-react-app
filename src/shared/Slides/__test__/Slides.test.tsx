
import React from "react";
import { shallow, mount } from "enzyme";
import * as renderer from "react-test-renderer";
import {Slides} from "./../index";

describe("Slides", () => {
  const defaultProps = {
    data: ['image-1','image-2'],
  };

  it("should render the Slides", () => {
    const subject = shallow(<Slides {...defaultProps} />);
    expect(subject).toHaveLength(1);
  });

  it("should render correctly and match to the Snapshot", () => {
    const subject = renderer.create(<Slides {...defaultProps} />).toJSON();
    expect(subject).toMatchSnapshot();
  });
});