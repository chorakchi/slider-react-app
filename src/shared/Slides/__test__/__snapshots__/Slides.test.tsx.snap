// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Slides should render correctly and match to the Snapshot 1`] = `
<div
  className="sc-bdnylx eyzPZC"
>
  <img
    className="sc-gtssRu itOySw"
    src="image-1"
  />
  <img
    className="sc-gtssRu itOySw"
    src="image-2"
  />
</div>
`;
